swagger.yaml

```diff
@@ -1634,6 +1634,52 @@ paths:
           description: Error.
           schema:
             $ref: '#/definitions/error'
+  /clusters/{cluster_id}/hosts/{host_id}/actions/reset:
```

Comment:
Isn't it part of reset cluster?
@avishayt do you think that we need to add another endpoint to reset day 2 hosts?

Reply:
@filanov , in day2 cluster is just a wrap around for hosts. each hosts is independent of another. there are not action on a cluster, only on individual host. One host maybe just installing, another discovering, and another just failed to install, and the cluster stays the same